app/src/Event/EventApi.php

```diff
@@ -208,6 +209,34 @@ public function unattend(EventEntity $event)
         throw new \Exception("Failed to unmark you as attending: " . $result);
     }
 
+
+    /**
+     * Get attendees for given event
+     * @param $attendees_uri
+     * @param $limit
+     * @param bool $verbose
+     * @return UserEntity[]
+     */
+    public function getAttendees($attendees_uri, $limit = 0, $verbose = false)
+    {
+
+        $attendees_uri .= "?resultsperpage={$limit}";
+        if ($verbose) {
+            $attendees_uri = $attendees_uri . '&verbose=yes';
+        }
+
```

Comment:
@Jean85 Don't worry too much about whitespace. If it doesn't break PSR-2, it's probably okay. The main thing to focus on is that the PR works as intended, especially for use-cases that maybe the submitter may not have thought to check.